packages/paste-core/components/combobox/__tests__/Combobox.spec.tsx

```diff
@@ -336,42 +344,38 @@ describe('Combobox', () => {
 
   describe('Controlled Combobox', () => {
     it('should be able to clear a Combobox', () => {
-      render(<ControlledCombobox />, {wrapper: ThemeWrapper});
+      const {getByTestId, getByRole, getAllByRole} = render(<ControlledCombobox />, {wrapper: ThemeWrapper});
       // open the combobox
-      fireEvent.click(screen.getByRole('textbox'));
+      fireEvent.click(getByRole('textbox'));
+      expect(getByRole('textbox').value).toEqual('');
       // select the first item
-      fireEvent.click(screen.getAllByRole('option')[0]);
+      fireEvent.click(getAllByRole('option')[0]);
+
       // @ts-ignore Property 'value' does not exist on type 'HTMLElement' (I get it, but this is right)
-      expect(screen.getByRole('textbox').value).toEqual('Andorra');
-      expect(screen.getByTestId('input-value-span').textContent).toEqual('"Andorra"');
-      expect(screen.getByTestId('selected-item-span').textContent).toEqual(
-        '{"code":"AD","label":"Andorra","phone":"376"}'
-      );
+      expect(getByRole('textbox').value).toEqual('Andorra');
```

Comment:
Observation: Where or not I agree, KCD suggests using `screen` as "the thing to do" https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen